README.md

```diff
@@ -1,2 +1,42 @@
 # definition-of-done
-Definition of Done for CorrelAid projects
+A Definition of Done for CorrelAid projects
+
+# What is a Definition of Done?
+
+Especially in volunteering contexts, it is important to have concrete achievements and moments of being "really done" with a task. 
+
+> Proverbially, to get an answer to that, the question to ask is, “I know that you are done, but are you DONE-done?”. ([Source](https://www.agilealliance.org/glossary/definition-of-done/))
+
+But how do we know that we're really done and have not forgotten anything important? At CorrelAid, we use a Definition of Done. 
+A Definition of Done (DoD) is a tool from Scrum, a popular framework that is often used in software development. It "creates transparency by providing everyone a shared understanding" of what "being done" means ([Scrum Guide](https://scrumguides.org/docs/scrumguide/v2020/2020-Scrum-Guide-US.pdf#zoom=100), p.12).
+
+We know that it can be hard to pay attention to things like code quality and documentation when your time resources are scarce - as is always the case with volunteering. However, we have found that things like good documentation and structure are a key factor for the long-term success and impact of CorrelAid projects. Broken or poorly documented code is very hard to use or adapt by NPO partners who are often coding beginners. Without a good README, it will be hard to impossible to for someone at CorrelAid to help out the NPO if a problem comes up a couple of months later when you're already off the project. And having old, irrelevant code or visualizations in your repository is probably very confusing for someone not familiar with the project. 
```

Comment:
```suggestion
We know that it can be hard to pay attention to things like code quality and documentation when your time resources are scarce - as is often the case with volunteering. However, we have found that things like good documentation and structure are a key factor for the long-term success and impact of CorrelAid projects. Broken or poorly documented code is very hard to use or adapt by NPO partners who are often coding beginners. Without a good README, it will be hard to impossible to for someone at CorrelAid to help out the NPO if a problem comes up a couple of months later when you're already off the project. And having old, irrelevant code or visualizations in your repository is probably very confusing for someone not familiar with the project. 
```